src/main/java/racingcar/OutputView.java

```diff
@@ -0,0 +1,34 @@
+package racingcar;
+
+import racingcar.car.Car;
+
+import java.util.List;
+
+public class OutputView {
+
+    public static void print(RacingGameResult result) {
+        System.out.println("실행 결과");
+        int numberOfTry = result.getRecordOfCar(0).size();
+        for (int timeLab = 0; timeLab < numberOfTry; timeLab++) {
+            printDashInTimeLab(result, timeLab);
+        }
+    }
+
+    private static void printDashInTimeLab(RacingGameResult result, int timeLab) {
+        int numberOfCars = result.getNumberOfCars();
+        List<Car> records = result.getRecords();
+
+        for (int carIndex = 0; carIndex < numberOfCars; carIndex++) {
```

Comment:
```suggestion
        for (Car car : records) {
```
index 를 꼭 알아야할 필요가 없는 경우
foreach 형태로 표현하면 좀 더 간결하게 표현이 가능하고 실수의 여지도 적어집니다!
https://wikidocs.net/264